src/templates/ssp/xml/FedRAMP-SSP-OSCAL-Template.xml

```diff
@@ -2079,16 +2080,19 @@
       <resource uuid="1a23a771-d481-4594-9a1a-71d584fa4123">
          <title>FedRAMP Master Acronym and Glossary</title>
          <prop name="type" value="fedramp-acronyms"/>
+         <prop ns="https://fedramp.gov/ns/oscal" name="type" value="fedramp-citations"/>
          <rlink href="https://www.fedramp.gov/assets/resources/documents/FedRAMP_Master_Acronym_and_Glossary.pdf"/>
          <remarks>
             <p>Must be present in a FedRAMP SSP.</p>
          </remarks>
       </resource>
       <resource uuid="a2381e87-3d04-4108-a30b-b4d2f36d001f">
+         <title>FedRAMP Logo</title>
          <description>
             <p>FedRAMP Logo</p>
          </description>
          <prop name="type" value="logo"/>
+         <prop ns="https://fedramp.gov/ns/oscal" name="type" value="fedramp-logo"/>
```

Comment:
Do we need `fedramp-logo` in addition to `logo`?

Reply:
I think so. There is a specific requirement that the FedRAMP logo appear in a FedRAMP SSP. There are other logos but the FedRAMP one has no other identificatio.

Reply:
And, there is no authoritative source for NIST RMF (as separate from FedRAMP) props, including one for a logo. We will likely have to create a set of values for NIST RMF, and another (superset) for FedRAMP.